src/tiled/gotodialog.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * %gotodialog.cpp%
```

Comment:
You are supposed to remove the `%` characters. :-)

The `%FILENAME%` is a placeholder that makes Qt Creator automatically fill in the file name in the license template (configured in Qt Creator > Options > C++ > File Naming).